src/deployment/deploy.py

```diff
@@ -217,15 +217,15 @@ def check_region(self):
 
     def create_password(self, object_id):
         # Work-around the race condition where the app is created but passwords cannot
-        # be created yet.
+        # be created yet. Timeout after 60 seconds.
         count = 0
         while True:
             time.sleep(5)
             count += 1
             try:
                 return add_application_password(object_id)
             except CLIError as err:
-                if count > 5:
+                if count > 12:
```

Comment:
Can this be updated to use a constant?  It would be useful when we come back later to understand why it's 12.

Reply:
Accepted and updated my code in the PR. Let me know if you had a different solution in mind. I'm open to suggestions.